### ISSUE TITLE
delve没有使用go-config配置的gopath(Did not use the gopath definition of go-config)

### ISSUE THREAD

**Issue by niuxiaozu** [lloiser/go-debug#59]
我通过修改源码使delve子进程使用go-config配置的gopath。可以不配置环境变量。但我可能考虑不全。
修改：lib/delve.js  
加入全局变量let goconfig
然后在337 338行 ，在main调用delve.get的时候为全局变量赋值
`export function get (goget, goconfig1) {
  goconfig = goconfig1;`
然后在run方法里 修改第60行语句
`dlvProcess = spawn(dlvPath, dlvArgs, {
    cwd: path.dirname(file),
    env: goconfig.environment()
  })`
这样就能不配置环境变量GOPATH 从而使用go-config配置的gopath了。好处了方便切换gopath

**Comment by lloiser** [comment]
Next time please use english even though google translate was able to translate it ... 

Ok so you want `go-debug` to use the `GOPATH` variable from the `go-config` setting.
That's already happening: I'm looking for the `dlv` executable using `goconfig` (see https://github.com/lloiser/go-debug/blob/master/lib/delve.js#L331)

So as long as you have no environment variable `GOPATH` (verify with `process.env.GOPATH` in the atom developer tools) and the `GOPATH` setting in `go-config` set you should get the correct `dlv`

**Comment by niuxiaozu** [comment]
Sorry, friends. What I mean is that when Delve runs, it needs the gopath variable to find the local library.
![1](https://cloud.githubusercontent.com/assets/7410862/19340020/5eabb892-9159-11e6-8410-23e058d52533.png)
I set the environment variable.
Then I debug main.go .
![2](https://cloud.githubusercontent.com/assets/7410862/19340124/e40c6e78-9159-11e6-85c1-a2f503477fdd.png)

main.go imported package "niuxz".
niuxz is the package I created in the workspace.
Then I delete the environment variable GOPATH.
I debug again.

![3](https://cloud.githubusercontent.com/assets/7410862/19340254/8385fee2-915a-11e6-9970-d6647724a830.png)

Delve throws such an exception. Means can not find package niuxz. The reason is that gopath is not configured. I think it should be a sub-process running delve not the environment variables.
I have modified the delve.js can achieve my needs. Your plugin is great and I like it.
Thank you for your work :-), and I am sorry that I am using the Chinese submission issue.
